content/welcome/_index.en.md

```diff
@@ -6,21 +6,27 @@ tags: ["wiki"]
 weight: 1
 ---
 
-This section describes how to get up and running!
+This page describes important first steps for getting started with R, RStudio, and the templates!
+
+A step-by-step walk-through of an outbreak template case study is available in the Training Materials pages of this website.
 
 ## NECESSARY COMPONENTS
 
 ### R
 
-The templates need at least R version 3.3. We recommend [getting R directly from CRAN](https://cran.r-project.org), but [the MRAN version of R](https://mran.microsoft.com/) will work as well.
+**R is a free software for statistical computing and graphics.** The templates need at least R version 3.3. We recommend [downloading R directly from CRAN](https://cran.r-project.org), a central repository of R code, packages, and documentation. If desired, [the MRAN version of R](https://mran.microsoft.com/) will work as well.
 
-> **Caveat for MRAN**: MRAN is a version of R that has modifications that are outside the scope of this project, however, it should be noted that it also locks the version of all packages to a specific date, so care is needed to specify repositories when installing R packages.
+> **Caveat if using MRAN**: MRAN is a version of R that has modifications that are outside the scope of this project, however, it should be noted that it also locks the version of all packages to a specific date, so care is needed to specify repositories when installing R packages. If you are unsure whether you have MRAN or CRAN R installed then see this [FAQ](<https://github.com/R4EPI/sitrep/wiki/4). 
```

Comment:
I've migrated the wiki on the github to this website so there is no need to link there. 

```suggestion
{{% notice info %}}
**Caveat if using MRAN**: MRAN is a version of R that has modifications that are outside the scope of this project, however, it should be noted that it also locks the version of all packages to a specific date, so care is needed to specify repositories when installing R packages. If you are unsure whether you have MRAN or CRAN R installed then see this [FAQ](https://r4epis.netlify.com/faq/#i-am-unsure-if-i-have-the-mran-or-cran-version-of-r-installed). 
{{% /notice %}}
```